client/gutenberg/extensions/simple-payments/edit.js

```diff
@@ -358,106 +356,100 @@ class SimplePaymentsEdit extends Component {
 			);
 		}
 
+		const Wrapper = isLoading ? Disabled : 'div';
```

Comment:
When the form is in a disabled state, toggle has some funky borders going on:

<img width="218" alt="image" src="https://user-images.githubusercontent.com/87168/48691485-620a2f80-ebdb-11e8-98e2-d56006553798.png">

Seems to be coming from Gutenberg core even in latest `master`; I'll open an issue there.

Meanwhile, we could fix it by setting `border: 0` ?

Reply:
Issue: https://github.com/WordPress/gutenberg/issues/12049